src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
+      self ! Continue
```

Comment:
by the way, the cassandra materialized view is eventually consistent and may take a little while to update, so it's good if you delay the sending of the notification (fits good together with the throttling, I think)